### PR TITLE
More detailed logging

### PR DESCRIPTION
**Pull request by alternatexp**
This makes the logs more readable and allows you to see what URL the victim requested.

### CODE REVIEW

File: wifiphisher/common/phishinghttp.py [hunk 1]
```diff
@@ -28,8 +28,8 @@ def get(self, url):
             self.render("index.html", **template.get_context())
             wifi_webserver_tmp = "/tmp/wifiphisher-webserver.tmp"
             with open(wifi_webserver_tmp, "a+") as log_file:
-                log_file.write('[' + T + '*' + W + '] ' + O + "GET " + T +
-                               self.request.remote_ip + W + "\n"
+                log_file.write('[' + T + '*' + W + '] ' + O + "GET" + W + " request from " + T +
```
Comment (blackHatMonkey) [inline wifiphisher/common/phishinghttp.py]: We use 99 characters as our maximum length so can you fix this line. All you have to do is move `self.request.path` to the next line.

**Comment by blackHatMonkey** [comment]
@alternatexp if you are too busy to make the changes I can do them myself.

File: wifiphisher/common/phishinghttp.py (hunk 1 above)
Comment (alternatexp) [inline wifiphisher/common/phishinghttp.py]: Didn't notice that, will do now!

**Comment by alternatexp** [comment]
I am homeless, I have all the time in the world :)

**Comment by alternatexp** [comment]
@blackHatMonkey All done!

**Comment by blackHatMonkey** [comment]
@alternatexp Thank you for your contribution :smile:.

**Pull request merged**